ads/yandex.js

```diff
@@ -23,7 +23,7 @@ const renderTo = 'yandex_rtb';
  * @param {!Object} data
  */
 export function yandex(global, data) {
-  validateData(data, ['blockId'], ['data', 'isAdfox']);
+  validateData(data, ['blockId'], ['data', 'onRender', 'onError']);
```

Comment:
Question: To confirm does this mean you no longer support `data-is-adfox` param, but instead expect `data-on-render` and `data-on-error` param from config? If so, please also update the change in documentation. Thanks

Reply:
yeah, no longer support for `data-is-adfox` param and new callbacks params instead.
Updated docs.